src/aggregate/aggregate_exec.c

```diff
@@ -521,7 +521,7 @@ static void sendChunk_Resp2(AREQ *req, RedisModule_Reply *reply, size_t limit,
       if (cursor_done) {
         RedisModule_Reply_LongLong(reply, 0);
         if (IsProfile(req)) {
-          req->profile(reply, req, has_timedout, req->qiter.err->reachedMaxPrefixExpansions);
+          req->profile(reply, req, has_timedout, req->qiter.err->reachedMaxPrefixExpansions, req->sctx->spec && req->sctx->spec->scan_failed_OOM);
```

Comment:
Not actually related to this PR, but can we just create `ProfilePrinterCtx` and move it here by value? The list of arguments to this function is getting long...